calc/HamiltonianTwo.cpp

```diff
@@ -551,10 +551,10 @@ void HamiltonianTwo::calculate(const Configuration &conf_tot) {
 
     std::cout << ">>TOT" << std::setw(7) << nSteps_two*symmetries.size() << std::endl;
 
-#pragma omp parallel for collapse(2) schedule(static, 1)
+#pragma omp parallel for schedule(static, 1)
```

Comment:
Why was collapsing removed?  Why is there the need for static scheduling?  If this is meant to impose some kind of ordering, don't parallelize altogether please.